src/growth.jl

```diff
@@ -48,9 +52,12 @@ end
     LogisticGrowth{R}(; rate, carrycap, timestep)
     LogisticGrowth{R,W}(; rate, carrycap, timestep)
 
-Logistic growth based on a growth rate layer, using exact solution.
+Logistic growth rate of population size N based on an intrinsic growth rate ``r`` and carry capacity ``K``, using the exact solution between timesteps ``t+1`` and ``t``:
 
-Saturation only applies with positive growth
+```
+N_{t+1} = (N_t K) / (N_t + (K - N_t) e^{-rt})
```

Comment:
You can use latex syntax for this:
https://juliadocs.github.io/Documenter.jl/stable/man/latex/#latex_syntax

Reply:
Done